spec/models/topic_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+  subject { create(:topic) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_uniqueness_of(:name) }
+  end
+
+  describe '#create' do
+    it 'creates the topic' do
+      previous_count = Topic.count
```

Comment:
I think you can change this for `expect { subject }.to change { Topic.count }.by(1)`